.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "parserOption": {
+  "parserOptions": {
```

Comment:
This change has already been merged. You need to rebase your PR
`got remote add upstream https://GitHub.com/freeCodeCamp/guides.git`
`git pull --rebase upstream master`

This will update the code you have locally and apply your changes again. If you have any issues, please reach out for help either here or in the contributors chat on gitter.